aten/src/THC/generic/THCTensorMathMagma.cu

```diff
@@ -210,7 +210,7 @@ THC_API void THCTensor_(geev)(THCState *state, THCTensor *re_, THCTensor *rv_, T
   magma_vec_t jobvr = jobvrs[0] == 'N' ? MagmaNoVec : MagmaVec;
   int n = a_->size[0];
```

Comment:
It would be better to fix the types here. `a_->size[0]` is already an `int64_t`. You can use `auto` or explicitly use the full-sized type.